src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java

```diff
@@ -463,6 +474,22 @@ public void siCliccaSulBottoneConfermaDelega() {
         }
     }
 
+    public void siCliccaSulBottoneAccettaDelega() {
+        logger.info("Si clicca su conferma del pop-up");
+
+        deleghePGPagoPAPage.clickBottoneConfermaDelega();
+    }
+
+    @And("Si controlla codice verifica accettazione delega errato")
+    public void siControllaCodiceVerificaAccettazioneDelegaErrato() {
+        logger.info("Si clicca su conferma del pop-up per controllo codice verifica errato");
+
+        deleghePGPagoPAPage.clickBottoneConfermaDelega();
+        if (this.deleghePGPagoPAPage.verificaEsistenzaErroreCodiceSbagliato()) {
```

Comment:
There is already a logger inside that warns you

Reply:
delete condition